Interface/AddOns/nMainbar/config.lua

```diff
@@ -17,10 +17,13 @@ nMainbar.Config = {
 
         hotkeyFontsize = 18,
         hotkeyFont = 'Interface\\AddOns\\nMainbar\\media\\font.ttf',
+
+        petHotKeyFontsize = 14,
+        petHotKeyFont = 'Interface\\AddOns\\nMainbar\\media\\font.ttf',
```

Comment:
Not convinced this is a useful option, why would anyone have different fonts on the petbar and the main actionbar?